tests/bismark/main.nf.test

```diff
@@ -49,12 +52,10 @@ nextflow_pipeline {
 
         then {
             assert workflow.success
-            assert snapshot(workflow.trace.tasks().size()).match()
-            // FIXME assert snapshot(workflow).match()
```

Comment:
Should there be a test here?

Reply:
this is the deleted portion

Reply:
or do you mean assert for trace.tasks().size() ?